hledger-lib/Hledger/Data/Amount.hs

```diff
@@ -639,13 +639,13 @@ isNegativeMixedAmount m =
 -- i.e. does it have zero quantity with no price, zero quantity with a total price (which is also zero),
 -- and zero quantity for each unit price?
 mixedAmountLooksZero :: MixedAmount -> Bool
-mixedAmountLooksZero = all amountLooksZero . amounts . normaliseMixedAmount
+mixedAmountLooksZero (Mixed ma) = all amountLooksZero ma
```

Comment:
While this certainly looks more efficient, couldn't it change the user-visible behaviour ? Eg with an amount like Mixed [usd 1, usd -1].

Reply:
Mixed amount is now normalised by design, so you cannot store `mixed [usd 1, usd -1]` in a way that differs from `mixed [usd 0]`.

Reply:
Ah, right. MixedAmount is now always normalised. (And probably there's no reason for normaliseMixedAmount to still be in the codebase..)

Reply:
Correct. We left it in to easy migration. Do you think we can take it out now? (Same for `mixedAmountSquashPricesForDisplay`)